### PR TITLE
The good stuff

### PR DESCRIPTION
**Pull request by rumblefrog**
notice me oliy

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ client.on('ready', () => {
 client.on('message', msg => {
   if (msg.author.bot)
       return;
+  
+  if (msg.author.id === '105122038586286080')
```
Comment (oliyy) [inline bot.js:14]: Who's id is this?

**Review by oliyy — CHANGES_REQUESTED** [review]

File: bot.js (hunk 1 above)
Comment (rumblefrog) [reply bot.js:14]: my (Fishy!)

Comment (oliyy) [reply bot.js:14]: Pfft, ruining the fun.

**Comment by oliyy** [comment]
No one escapes the pasta enforcement team

**Pull request closed without merging**